### PR TITLE
Add Nyx chain 

### PR DESCRIPTION
**Pull request by mfahampshire**

### CODE REVIEW

**Comment by mfahampshire** [comment]
I am having an issue running the `yarn validate` command: when I remove `stakeCurrency` from the file, the command errors because this field is misssing. However when I introduce it (as I have in this recent commit) then the validation fails claiming that there is an unknown field in the file. 

Any advice on this would be very much appreciated.

File: cosmos/nyx.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    "bech32PrefixAccPub": "npub",
+    "bech32PrefixValAddr": "nvaloper",
+    "bech32PrefixValPub": "nvaloperpub",
+    "bech32PrefixConsAddr": "nvalcons",
+    "bech32PrefixConsPub": "nvalconspub"
+  },
+  "stakeCurrency": {
+    "coinDenom": "nyx",
+    "coinMinimalDenom": "unyx",
+    "coinDecimals": 6
```
Comment (HeesungB) [inline cosmos/nyx.json:21]: Is it right coin to stake?

File: cosmos/nyx.json [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    }
+  ],
+  "coinType": 118,
+  "gasPriceStep": {
+      "low": 0.01,
+      "average": 0.025,
+      "high": 0.03
+  },
+  "features": [ 
+      "cosmwasm", "ibc-transfer"
```
Comment (HeesungB) [inline cosmos/nyx.json:46]: Could you remove "ibc-transfer" field?

**Review by HeesungB — COMMENTED** [review]
You have to add node provider field. like under example

```
"nodeProvider": {
    "name": "Blockapsis",
    "email": "infra@blockapsis.com",
    "website":"https://blockapsis.com/"
  },
  ```

File: cosmos/nyx.json (hunk 1 above)
Comment (mfahampshire) [reply cosmos/nyx.json:21]: Yes, this is the coin that will be staked on the blockchain - `NYM` token is only used for the mixnet.

File: cosmos/nyx.json [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+        "coinDecimals": 6,
+        "coinGeckoId": "nym"
+    }
+  ],
+  "coinType": 118,
+  "gasPriceStep": {
+      "low": 0.01,
+      "average": 0.025,
+      "high": 0.03
+  },
```
[the remark this answers is not in the document]
Comment (mfahampshire) [reply cosmos/nyx.json]: removed

File: cosmos/nyx.json (hunk 2 above)
Comment (mfahampshire) [reply cosmos/nyx.json]: removed

**Comment by HeesungB** [comment]
@mfahampshire Thank you for contribution

**Pull request merged**